db/schema.rb

```diff
@@ -55,15 +55,16 @@
     t.boolean "call_present"
     t.date "call_open_date"
     t.date "call_close_date"
-    t.string "intended_beneficiaries", array: true
     t.string "roda_identifier_fragment"
     t.string "roda_identifier_compound"
+    t.string "intended_beneficiaries", array: true
```

Comment:
I always forget if we are supposed to worry about these lines shifting in the schema?

Reply:
I've got rid of it, just in case. Noticed it jumping back on me, so let's keep it where it is